### ISSUE TITLE
Greater "Portal" UX

### ISSUE THREAD

**Issue by Bjorn248** [codegnv/business-web#14]
Find Dave's picture below

![start-a-business](https://cloud.githubusercontent.com/assets/1063707/18687551/9afaa9e2-7f4e-11e6-9b40-d91a7e6d0c32.jpg)

**Comment by samanthawolfe** [comment]
Workflow and links to resources https://docs.google.com/document/d/1Sjo6M_F81wyHLzpaXwRcxQNgfSk4WaH6pyM0Nw8aJ0A/edit?usp=sharing

**Comment by gotoplanb** [comment]
- Each page is going to just be a components with some content.
- Nav bar is like trailhead with circle and active page larger circle
- Splash page is big image of a store, "open for business", and CTA to the first step.
- Samantha has all the content

**Comment by samanthawolfe** [comment]
I've talked further with Department of Doing folks. Here is something created by IDEO to explain the starting and growing your business workflow. https://www.dropbox.com/home/User-Centered%20Gainesville/04%20-%20deliverables/04%20Final%20Presentation%20Materials/Service%20BluePrint?preview=Service+BluePrint.pdf
DoD is going to create paper visuals for distribution and would like the biz portal workflow to mirror this as much as possible. I am going to edit the google doc to resemble the language. Can you work with this for visuals?

**Comment by samanthawolfe** [comment]
Screenshot of specific reference: 
![image](https://cloud.githubusercontent.com/assets/19419412/23171746/6707ee32-f821-11e6-9040-1daf8b8a90dc.png)

**Comment by samanthawolfe** [comment]
Here's a complete dropbox folder which may contain visual assets needed... https://www.dropbox.com/home/User-Centered%20Gainesville/04%20-%20deliverables/04%20Final%20Presentation%20Materials/Service%20BluePrint

**Comment by gotoplanb** [comment]
@samanthawolfe the Dropbox link is broken for me

**Comment by samanthawolfe** [comment]
@gotoplanb Try this - https://www.dropbox.com/sh/afm383ob9lkr7p1/AADtPBYY06X_Pxq-g5phdBPGa?dl=0

**Comment by gotoplanb** [comment]
@samanthawolfe Did any other conversation happen after I left that would invalidate my wireframes? We can later change the number of navigation items and aesthetic, but at the core are we still good with:

* Creating a global header
* Creating a global navigation bar
* Creating a content-only page template to be used for all of the other business phases for now?
* Create routes for all the content pages
* Add the global header and global navigation above the currently implemented permit locator

Based on @Bjorn248's feedback related to search prominence, the permit-locator page should keep the page-level header (underneath global navigation) to have both the search field and the helper text (as currently built)

Just wanted to verify with you before I created the additional tickets.

![portal](https://cloud.githubusercontent.com/assets/51855/23217042/f6ff13d4-f8e5-11e6-805a-d68d6012c825.JPG)

**Comment by Bjorn248** [comment]
@gotoplanb I believe that @hollymariegibbs wants to push for a material design style table of contents, as seen here [https://material.io/guidelines/](https://material.io/guidelines/) when you click on the hamburger menu, having them as a numbered list to display order, and also having it always be visible. I don't have too many strong feelings about which one we choose, but the table of contents approach doesn't seem to be like a path / progress bar as much as a top bar does, however, the top bar might not feel as intuitive as the table of contents approach? I'm not sure I can talk too much about design in the presence of you two (credentialed design folks).

**Comment by gotoplanb** [comment]
Reassigned to @hollymariegibbs to take it from here. I won't create any tickets yet for content pages or global header+nav.

**Comment by hollymariegibbs** [comment]
I am working on mockups of this all. Have a meeting tomorrow with @samanthawolfe, the Buda team and Lila (Strategic Planning Manager) to iron out details and put together a plan for moving forward. This has significant integration with the Department of Doing's site and process, so we want to make sure we're moving in the right direction. Will report afterward.

**Comment by Bjorn248** [comment]
@hollymariegibbs didn't want to create a duplicate of this issue, but this is the other issue we're waiting on UX for. @ajcrites already did some work regarding this in #129 that's a good starting point for the designs provided by @gotoplanb. Just need official word on this so we know exactly what the city expects from us here.

**Comment by gotoplanb** [comment]
:shipit:
